frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -89,6 +89,11 @@ const AdminJobsPage = () => {
         MilkTheCowsMutation.mutate();
     }
 
+    const submitInstuctorReportJob = async () => {
+        console.log("submitInstructorReportJob (wip)");
+        toast('Submitted Job: Instructor Report (wip)');
```

Comment:
Can we change this to:
```suggestion
        toast('Instructor report not yet implemented; coming soon');
```